internal/ingress/types.go

```diff
@@ -64,6 +64,9 @@ type Configuration struct {
 	// +optional
 	PassthroughBackends []*SSLPassthroughBackend `json:"passthroughBackends,omitempty"`
 
+	// ConfigmapChecksum contains the particular checksum of a Configuration object
```

Comment:
As discussed on Slack, I find this name confusing and would have used `BackendConfigChecksum` instead.